coldfront/core/utils/views/mou_views.py

```diff
@@ -45,14 +36,17 @@ def dispatch(self, request, *args, **kwargs):
     def set_attributes(self, pk):
         """Set this instance's request_obj to be the
         SavioProjectAllocationRequest with the given primary key."""
-        self.request_type = self.kwargs['mou_type']
-        self.request_type_long, self.request_class = \
+        self.request_type = self.kwargs['request_type']
+        self.request_type_long, self.request_class, self.mou_type = \
```

Comment:
This line is hard to read. Define the `dict` first, and then set these three values?